assets/templates/kubernetes.container/schema-b/configs.yml

```diff
@@ -33,11 +33,30 @@ fields:
     range:
       min: 0
       max: 10000
-  - name: Percentage
+  - name: PercentageMemory
     range:
       min: 0
-      max: 5000
+      max: 100
     fuzziness: 0.005
+  - name: PercentageCPU
+    range:
+      min: 0
+      max: 100
+    fuzziness: 0.005
```

Comment:
@gizas 
there's a known tendency towards generating decreasing values up to the minimum one when providing `fuzziness`: https://github.com/elastic/elastic-integration-corpus-generator-tool/issues/127

Coupled with an integer type field and a range with `min: 0` this tendency ends up generating nothing else other than `0` after a certain amount of events.

With a double type field, like `PercentageCPU`, the tendency is still present, but thanks to no need to round a float to an integer, internally, there's probably a lower chance to generate just a series of `0` after a while.
In any case I'd suggest to increase `min` to something just a little bigger than `0`, `0.1` can be a good start, but I'll leave to you to proper tweak a better value.

Still after a certain amount of events the values generated will "float" around `0.1`, or whatever value you set as `range.min`

Reply:
Thanks !!! I kept the below suggestion so it is not needed